### ISSUE TITLE
Feature Request: Chat Integration

### ISSUE THREAD

**Issue by azophy** [monicahq/monica#68]
I'm actually still in the process of setting up Monica on my machine, but from what I read in the Readme, it only mentioned that the means for communicating events to user is by email. Rather than email, I think making chatbot would make this software more accessible to many people (for instance, its more common for people from my country to access chat apps rather then their email).

From what I know, implementing chatbot notification is rather easy nowadays. From what I have tried in Telegram, its only a matter of implementing some simple REST API call. I think this is pretty similiar for Slack or LINE (I don't actually have any experience in both apps, but I think it should be).

Nonetheless, many appreciations for all Monica Contributors!

**Comment by Kilian** [comment]
I would love this! Gentle nudges via chat to send a message to people is such a low-friction way to help people keep in touch. If there is a Monica API (like in the project TODO), I wouldn't mind taking a stab at a facebook messenger chatbot.

**Comment by degan6** [comment]
This would be awesome I agree.

Laravel has notifications are designed just for this. Send a text, slack, webhook and so on.   

I don't think the support FB messenger but I'm sure they could. 

[https://laravel.com/docs/5.4/notifications](https://laravel.com/docs/5.4/notifications)

**Comment by djaiss** [comment]
Yes. 100x times yes. I'd love to have Telegram notifications personally, but it could be to Slack or whatever.

**Comment by tomzx** [comment]
Probably of interest: http://laravel-notification-channels.com/
